rclpy/src/rclpy/_rclpy.c

```diff
@@ -3754,7 +3755,7 @@ rclpy_get_rmw_qos_profile(PyObject * Py_UNUSED(self), PyObject * args)
   } else if (0 == strcmp(pyrmw_profile, "qos_profile_parameter_events")) {
     pyqos_profile = rclpy_common_convert_to_qos_dict(&rmw_qos_profile_parameter_events);
   } else {
-    PyErr_Format(PyExc_RuntimeError,
+    PyErr_Format(RCLError,
```

Comment:
Not an error from `rcl`.
```suggestion
    PyErr_Format(PyExc_RuntimeError,
```